diff -u a/lib/Kconfig b/lib/Kconfig
--- a/lib/Kconfig
+++ b/lib/Kconfig
@@ -328,21 +328,27 @@
 config LZO_DECOMPRESS
	tristate

 config LZ4_COMPRESS
	tristate

+config LZ4K_COMPRESS
+	tristate
+
 config LZ4HC_COMPRESS
	tristate

 config LZ4KD_COMPRESS
	tristate

 config LZ4_DECOMPRESS
	tristate

+config LZ4K_DECOMPRESS
+	tristate
+
 config LZ4KD_DECOMPRESS
	tristate

 config ZSTD_COMMON
	select XXHASH
	tristate
 
